.gitmodules

```diff
@@ -16,3 +16,12 @@
 [submodule "33-room-occupancy-monitor/firmware/note-c"]
 	path = 33-room-occupancy-monitor/firmware/note-c
 	url = https://github.com/blues/note-c
+[submodule "33-room-occupancy-monitor/firmware/sparrow-lora"]
```

Comment:
This looks like a duplicate.

Reply:
yup, I did see it got forgot to remove it. duh!